tests/hash-graph-benches/representative_read/seed.rs

```diff
@@ -327,9 +327,10 @@ pub async fn setup_and_extract_samples<A: AuthorizationApi>(
     store_wrapper: &mut StoreWrapper<A>,
 ) -> Samples {
     // TODO: We'll want to test distribution across accounts
-    //  https://app.asana.com/0/1200211978612931/1203071961523000/f
-    // We use a hard-coded UUID to keep it consistent across tests so that we can use it as a
-    // parameter argument to criterion and get comparison analysis
+    //   See https://linear.app/hash/issue/H-3012
+    //
+    //   We use a hard-coded UUID to keep it consistent across tests so that we can use it as a
+    //   parameter argument to criterion and get comparison analysis
```

Comment:
```suggestion
    //   see https://linear.app/hash/issue/H-3012
    //
    // We use a hard-coded UUID to keep it consistent across tests so that we can use it as a
    // parameter argument to criterion and get comparison analysis
```